db/schema.rb

```diff
@@ -480,6 +481,12 @@
     t.datetime "updated_at",  null: false
   end
 
+  create_table "url_exclusions", force: :cascade do |t|
+    t.string   "url_substring"
+    t.datetime "created_at",    null: false
+    t.datetime "updated_at",    null: false
+  end
+
```

Comment:
I'm not sure how this snuck in but it shouldn't be in the schema. We don't use this table.

Reply:
🤷‍♂